HARK/abstract/tests/consindshk_full.yml

```diff
@@ -0,0 +1,52 @@
+---
+states: !StateSpace
+  variables:
+    - !State
+      name: m
+      short_name: money
+      long_name: market resources
+      latex_repr: \mNrm
+    - !State
+      name: &name stigma
```

Comment:
I don't understand what the ampersands are doing here.
Maybe document that?

Reply:
ampersands are aliases, since one of the states is also a control and a post-state. I will document this more as I make more progress.